package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@yext/search-core",
-  "version": "2.6.0-beta.3",
```

Comment:
Is this right? 2.6.0 doesn't match the title 2.5.3

Reply:
Yes, because now that 2.5.3 is on master, we want to pull it into develop, where 2.6 is being developed, so this becomes a new patch of the 2.6 beta